modules/extensions-support/src/main/java/org/mule/module/extension/internal/util/NameUtils.java

```diff
@@ -155,13 +157,30 @@ public static String camel(String hyphenizedValue)
     }
 
     /**
-     * Transforms a camel case value into a hyphenizedone.
+     * Transforms a camel case value into a hyphenated one.
+     * <p>
+     * For example:
+     * {@code messageProcessor} would be transformed to {@code message-processor}
+     *
+     * @param camelCaseNames an array {@link String} in camel case form
+     * @return the {@code camelCaseNames} hyphenated and concatenated in a hyphenated form
+     */
+    public static String hyphenize(String... camelCaseNames)
```

Comment:
you probably need to revert the changes on this class because of the "as" going away